### DISCUSSION TITLE
Losses stay stable, model does not improve?

### DISCUSSION THREAD

**Discussion by horsto** [facebookresearch/detectron2#4630] [Q&A]
I am training `keypoint_rcnn_R_50_FPN_3x` (I also tried R101), and my total losses and individual (sub) losses remain stable after ~3000 iterations and the model predictions seem quite noisy still. As a beginner I am wondering: Does that mean that my config should be tweaked (e.g. learning rate should be lower, ...) or that I do not have enough training data or that the model I have chosen is not up to the job?
